src/main/java/at/hannibal2/skyhanni/features/inventory/SoulboundItemIcon.kt

```diff
@@ -0,0 +1,55 @@
+package at.hannibal2.skyhanni.features.inventory
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.GuiRenderItemEvent
+import at.hannibal2.skyhanni.events.LorenzToolTipEvent
+import at.hannibal2.skyhanni.utils.ItemUtils.getLore
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.RenderUtils.drawSlotText
+import at.hannibal2.skyhanni.utils.StringUtils.matchMatcher
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class SoulboundItemIcon {
+    val config get() = SkyHanniMod.feature.inventory.soulbound
+    val coopSoulboundPattern by RepoPattern.pattern(
+        "inventory.coopsoulbound",
+        "§8§l* §8Co-op Soulbound §8§l*"
+    )
+    val soulboundPattern by RepoPattern.pattern(
+        "inventory.soulbound",
+        "§8§l* §8Soulbound §8§l*"
+    )
```

Comment:
These should be part of a pattern group. Maybe "inventory.item.soulbound"

Then first pattern is "coop" and second is "player"

Also missing an empty line after the patterns